MapboxNavigation/RouteMapViewController.swift

```diff
@@ -801,25 +802,56 @@ extension RouteMapViewController: NavigationViewDelegate {
                 if minDistanceBetweenPoints < smallestLabelDistance {
                     smallestLabelDistance = minDistanceBetweenPoints
                     
-                    if let line = feature as? MGLPolylineFeature, let name = line.attribute(forKey: "name") as? String {
-                        currentName = name
-                    } else if let line = feature as? MGLMultiPolylineFeature, let name = line.attribute(forKey: "name") as? String {
-                        currentName = name
-                    } else {
-                        currentName = nil
+                    if let line = feature as? MGLPolylineFeature {
+                        if let name = line.attribute(forKey: "name") as? String {
+                            currentName = name
+                        } else if let line = feature as? MGLMultiPolylineFeature, let name = line.attribute(forKey: "name") as? String {
```

Comment:
This line smells suspicious to me. It's existing logic that checks for `line = feature as? MGLPolylineFeature` and `line = feature as? MGLMultiPolylineFeature` within its scope. Needs further investigation why this exists.
/cc @bsudekum @1ec5